### DISCUSSION TITLE
Regarding WebHook

### DISCUSSION THREAD

**Discussion by Rushithab** [postalserver/postal#2896] [Development discussions]
I have result of all the sent emails. How to get all those email information data in webhooks? Couldn't find proper documentation on this. 
Thanks

**Comment by willpower232** [comment]
Webhooks are sent when a message status changes to keep you up to date on what really happened to it, there can be a delay after the message is sent as it gets queued and processed later.

https://docs.postalserver.io/developer/webhooks

What information are you looking for exactly?

**Comment by Rushithab** [comment]


**Reply by willpower232** [reply]
That page already lists all the events and their JSON payloads. Which bit do you need extra help with?
